tasks/cops_documentation.rake

```diff
@@ -0,0 +1,284 @@
+# frozen_string_literal: true
+
+require 'yard'
+require 'rubocop'
+require 'rubocop-rails'
+
+YARD::Rake::YardocTask.new(:yard_for_generate_documentation) do |task|
+  task.files = ['lib/rubocop/cop/**/*.rb']
+  task.options = ['--no-output']
+end
+
+desc 'Generate docs of all cops departments'
```

Comment:
"cop's"?

Reply:
»cops’«, if anything. But I think cops (without apostrophe) is fine.

Reply:
Thanks for the review. This time I'd like to describe it according to RuboCop RSpec.
https://github.com/rubocop-hq/rubocop-rspec/blob/v1.30.1/tasks/cops_documentation.rake#L12

Reply:
It looks like we copied it from RuboCop itself: 
https://github.com/rubocop-hq/rubocop/blob/f2ec4a6ce020b8874593f658c71fb11e17e53edc/tasks/cops_documentation.rake#L11